hushline/model.py

```diff
@@ -297,3 +307,106 @@ def __init__(self) -> None:
 
     def __repr__(self) -> str:
         return f"<InviteCode {self.code}>"
+
+
+# Paid tiers
+class Tier(Model):
+    __tablename__ = "tiers"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    name: Mapped[str] = mapped_column(db.String(255), unique=True)
+    monthly_amount: Mapped[int] = mapped_column(db.Integer)  # in cents USD
+    stripe_product_id = mapped_column(db.String(255), unique=True)
+    stripe_price_id = mapped_column(db.String(255), unique=True)
+
+    def __init__(self, name: str, monthly_amount: int) -> None:
+        super().__init__()
+        self.name = name
+        self.monthly_amount = monthly_amount
```

Comment:
It looks like we're defining a constructor to require `name` and `monthly` amount which not allowing other columns to be specified on creation. In that case, we can still make use of the existing constructor:
```suggestion
        super().__init__(name=name, monthly_amount=monthly_amount)
```